.env

```diff
@@ -1,5 +1,12 @@
 VIRTUAL_HOST=localhost
 USE_TLS=false
+INTEGRATED_TURN_PORT_RANGE=50000-50050
+INTEGRATED_TURN_IP=192.168.83.130
+STORE_METRICS=true
+DB_PASSWORD=postgres
+DATABASE=membrane_rtc_engine_timescaledb_test 
+DB_USERNAME=postgres
+DB_HOSTNAME=localhost
```

Comment:
Do we need to have these variables set in .env?

Reply:
no :)